### ISSUE TITLE
XsdBasedSoap11Wsdl4jDefinitionBuilder throws BeanCreationException while generating wsdl [SWS-601]

### ISSUE THREAD

**Issue by gregturn** [spring-projects/spring-ws#717]
**[Prasad](https://jira.spring.io/secure/ViewProfile.jspa?name=regulaprasad)** opened **[SWS-601](https://jira.spring.io/browse/SWS-601?redirect=false)** and commented

While I was trying to generate it is throwing an exception. I am pasting the exception below

org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'echo' defined in ServletContext resource [/WEB-INF/spring-ws-servlet.xml]: Cannot create inner bean 'org.springframework.ws.wsdl.wsdl11.builder.XsdBasedSoap11Wsdl4jDefinitionBuilder#1498436' of type [org.springframework.ws.wsdl.wsdl11.builder.XsdBasedSoap11Wsdl4jDefinitionBuilder] while setting bean property 'builder'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.ws.wsdl.wsdl11.builder.XsdBasedSoap11Wsdl4jDefinitionBuilder#1498436' defined in ServletContext resource [/WEB-INF/spring-ws-servlet.xml]: Invocation of init method failed; nested exception is org.xml.sax.SAXParseException: The processing instruction target matching "[xX][mM][lL]" is not allowed.

The bean definition in my servlet is as below
\<bean id="echo" class="org.springframework.ws.wsdl.wsdl11.DynamicWsdl11Definition">  
\<property name="builder">  
\<bean  
class="org.springframework.ws.wsdl.wsdl11.builder.XsdBasedSoap11Wsdl4jDefinitionBuilder"> //Error at this line  
\<property name="schema" value="/WEB-INF/echo.xsd"/>  
\<property name="portTypeName" value="Echo"/>  
\<property name="locationUri" value="http://localhost:8080/echoservice/"/>  
\</bean>  
\</property>  
\</bean>

---

**Affects:** 1.5.8

**Comment by snicoll** [comment]
I don't believe this one is actionable, this feels like some local modification to the sample.
